Dockerfiles/Dockerfile.bdt

```diff
@@ -2,10 +2,13 @@ FROM node:current-slim
 
 WORKDIR '/'
 
-RUN apt update -y && apt install curl jq git -y
```

Comment:
Would be cool to see what dependencies/tools are available in a base image!

Reply:
Yup!  The base image optimization strategy is going to be fun!